api/internal/core/entity/entity.go

```diff
@@ -97,10 +97,11 @@ type Route struct {
 }
 
 // --- structures for upstream start  ---
+type TimeoutValue float32
```

Comment:
why

Reply:
There are two "number" definitions in jsonschema, one is an `integer`, the other is `number`, where `number` runs to store float values. Such as 5.5

Therefore, a float value is used here to store.

In addition, only map[string]interface{} was used to store the value before, and there was ambiguity in the data type.  Crash may occur during runtime.  This is also the reason for this modification.（add a Timeout struct）

Reply:
cc @nic-chen @starsz

Reply:
so do we support timeout 5.5s?

Reply:
![image](https://user-images.githubusercontent.com/2561857/131994988-a6f630d3-709d-4ecb-9072-0607a89d3ae2.png)

It seems that `float` is ok